requirements.txt

```diff
@@ -3,8 +3,8 @@ attrdict==2.0.0
 eventlet==0.19.0
 python-socketio==1.6.1
 numpy==1.13.1
-Pillow==2.2.1
+Pillow==6.1.0
 scipy==0.19.1
 keras==2.0.8
-tensorflow==1.3.0
+tensorflow==1.14.0
```

Comment:
Is is a good idea, to migrate tensorflow library from 1.3 to 1.14? This post is dealing with this question:
https://knowledge.udacity.com/questions/27057

Although if we don't want to test our code on Carla, maybe doesn't matter.